docs/serving/samples/hello-world/helloworld-go/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/knative/docs/docs/serving/samples/hello-world/helloworld-go
+
+go 1.13
```

Comment:
Missed an empty line after this line. Otherwise `go build` will not work with `-mod=readonly`.